File: Service_eHospital_DongNai_A/Implement/zzz_NgoaiTru_XuatLaiXMLService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class zzz_NgoaiTru_XuatLaiXMLService : BaseService<zzz_NgoaiTru_XuatLaiXML, Izzz_NgoaiTru_XuatLaiXMLRepository>
    , Izzz_NgoaiTru_XuatLaiXMLService
    {
    private readonly Izzz_NgoaiTru_XuatLaiXMLRepository _zzz_NgoaiTru_XuatLaiXMLRepository;
    public zzz_NgoaiTru_XuatLaiXMLService(Izzz_NgoaiTru_XuatLaiXMLRepository zzz_NgoaiTru_XuatLaiXMLRepository) : base(zzz_NgoaiTru_XuatLaiXMLRepository)
    {
    _zzz_NgoaiTru_XuatLaiXMLRepository = zzz_NgoaiTru_XuatLaiXMLRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/zzz_NgoaiTru_XuatLaiXML.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { zzz_NgoaiTru_XuatLaiXML } from './zzz_NgoaiTru_XuatLaiXML.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class zzz_NgoaiTru_XuatLaiXMLService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'STT', 'SoTiepNhan'];

    List: zzz_NgoaiTru_XuatLaiXML[] | undefined;
    ListFilter: zzz_NgoaiTru_XuatLaiXML[] | undefined;
    FormData!: zzz_NgoaiTru_XuatLaiXML;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "zzz_NgoaiTru_XuatLaiXML";
    }
}

